CHANGES.md

```diff
@@ -18,12 +18,19 @@ OpenSSL Releases
  - [OpenSSL 1.0.1](#openssl-101)
  - [OpenSSL 1.0.0](#openssl-100)
  - [OpenSSL 0.9.x](#openssl-09x)
-
+)
```

Comment:
Oh...wait...what is this close paren doing here?

Reply:
I think I was distracted by the heart attack that this file seems to give my editor. Fixed now.

Reply:
Oh c'mon, you're kiddin' me! The (minimalistic) markdown was designed to not only look awesome in the browser but also to remain readable in raw ASCII format, see #10545.  At least I tried very hard to keep my promise.

Reply:
Perhaps running markdownlint on all ```.md``` files would be a fun project for someone to add to the CI's?